app/models/work.rb

```diff
@@ -401,7 +401,18 @@ def post_curation_s3_resources
 
     s3_resources
   end
-  alias post_curation_uploads post_curation_s3_resources
+
+  # Returns the files in post-curation for the work
```

Comment:
@hectorcorrea I would have expected post_curation_uploads to force the correct state by default.  It seems you have found and fixed a bug.  Would it be possible to change the default to true?

Reply:
Yeah, it could be but I didn't want to change the existing behavior on this PR. There are several refactors that we should do to the Work as it relates to the fetching of pre/post curation files, we have way too many methods that do the same/similar things on this class and I am not sure it is the best approach, but I didn't want to include that work on this PR.

Reply:
New issue for refactor: https://github.com/pulibrary/pdc_describe/issues/1192